docs/config-file/v2.rst

```diff
@@ -319,8 +319,8 @@ The Docker image used for building the docs.
 Each image support different Python versions and has different packages installed,
 as defined here:
 
-* `stable <https://github.com/readthedocs/readthedocs-docker-images/tree/releases/4.x>`_: :buildpyversions:`stable`
-* `latest <https://github.com/readthedocs/readthedocs-docker-images/tree/releases/5.x>`_: :buildpyversions:`latest`
+* `stable <https://github.com/readthedocs/readthedocs-docker-images/tree/releases/5.x>`_: :buildpyversions:`stable`
```

Comment:
I feel like we should probably have a mapping for these somewhere, so we only have to change them in 1 place? Not sure the best way to handle this, but it seems we have already built some kind of sphinx extension here to support some of this?